source/includes/_product_data.md

```diff
@@ -421,7 +506,7 @@ name | description | default
  ----:|:--- | ---
 `start` | start-time | 7 days ago
 `end` | end-time | now()
-`fields` | fields to query | ["current", "voltage", "temperature"]
+`fields` | fields to query | ["*"]
```

Comment:
I think don't include the brackets. Also put `(all fields)` afterwards to make it clear